s2a/src/test/resources/config.cnf

```diff
@@ -0,0 +1,17 @@
+[req]
+distinguished_name = req_distinguished_name
+req_extensions = req_ext
+
+[req_distinguished_name]
```

Comment:
Could you add the values you entered while generating certs?

Reply:
Done, as discussed offline, I updated the README with the the values used to populate these fields. 

@ejona86, does this sound ok to you? @erm-g pointed me to something similar he has done in go: https://github.com/grpc/grpc-go/pull/6670/files#diff-ac85255b49c8d65b03b560d9959b99c6d9338de1400da6fc2d4133a434a376ad